### PR TITLE
Fix an issue with index

### PR DESCRIPTION
**Pull request by natsheh**

### CODE REVIEW

**Comment by natsheh** [comment]
@lmartinet for your review

File: utils/load_corpus.py [hunk 1]
```diff
@@ -62,15 +61,11 @@ def __iter__(self):
 					words_lst = self.tokenize(u_line)
 					try :
 						self.yeild_element(words_lst, tag=self.istex_tag, count=self.paper_count)
```
Comment (lmartinet) [inline utils/load_corpus.py]: Misspelled function ! yield.element()

Comment (lmartinet) [inline utils/load_corpus.py]: Misspelled function ! yield_element()

File: utils/load_corpus.py [hunk 2]
```diff
@@ -176,5 +169,5 @@ def yield_element(self, words_lst, tag=None, count=None):
 			yield words_lst
 
 	def count(self):
```
Comment (lmartinet) [inline utils/load_corpus.py]: We can remove this function, now.

**Review by lmartinet — CHANGES_REQUESTED** [review]

**Pull request merged**